resources/views/components/breadcrumbs.blade.php

```diff
@@ -6,7 +6,7 @@
         @foreach($breadcrumbs as $key=>$crumb)
             @if($key == 0)
                 <li class="inline">
-                    <a href="/"><span class="text-black align-middle">@svg('home', 'w-4 h-4 inline align-baseline')</span></a>
+                    <a href="/" aria-labelledby="Home"><span class="text-black align-middle">@svg('home', 'w-4 h-4 inline align-baseline')</span></a>
```

Comment:
I believe this should be `aria-label` per https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute unless you're planning to point to an `id` on the page.

Reply:
Ah yes, that is correct. I didn't look into the SVG to see the `id` there. Looking though it is, `home` but the capitalization of `Home` threw me off.

Can you change this to match the ID? `aria-labelledby="home"`

Reply:
Will do in the next PR!